lib/src/utils.dart

```diff
@@ -26,8 +26,13 @@ Future<ProcessResult> runProc(
   String? workingDirectory,
   Map<String, String>? environment,
   Duration? timeout,
-  bool deduplicate = false,
```

Comment:
Is this considered publicly exposed? Otherwise I would just remove it straight away.

Reply:
With prior breaking changes we were honoring the semantic versioning, and if we want to remove this, I'd rather bump the version. I'm fine either way.